app/templates/colors/index.html

```diff
@@ -3,16 +3,34 @@
 <head>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Rainbow Translator!</title>
- <script src="https://code.jquery.com/jquery-3.2.1.js"
+  <script src="https://code.jquery.com/jquery-3.2.1.js"
           integrity="sha256-DZAnKJ/6XZ9si04Hgrsxu/8s717jcIzLy3oi35EouyE="
           crossorigin="anonymous"></script>
   <script type="text/javascript"
           src="//cdnjs.cloudflare.com/ajax/libs/socket.io/1.3.6/socket.io.min.js"></script>
-    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
-          rel="stylesheet">
-    <script type="text/javascript" src="{{ url_for('static', filename = 'app.js') }}"></script>
-    <link rel="stylesheet" href="{{ url_for('static', filename = 'main.css') }}"/>
-    <link rel="shortcut icon" href="{{ url_for('static', filename='favicon.ico') }}">
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/mustache.js/2.3.0/mustache.min.js"></script>
```

Comment:
Albo wszędzie używamy `//cdnjs. ...` albo wszędzie `https://cdnjs. ...`
Tutaj piszą że najlepiej używać `//` żeby przeglądarka użyła tego samego protokołu którego użyła do pobrania htmla, jednak z drugiej strony `https` nie daje dużo narzutu.
https://stackoverflow.com/questions/4363669/advantages-of-cdn-content-served-via-https-over-http